packages/node_modules/@webex/react-component-activity-item-base/src/index.test.js

```diff
@@ -108,4 +109,28 @@ describe('ActivityItemBase component', () => {
 
     expect(component).toMatchSnapshot();
   });
+
+  it('renders properly when message is edited', () => {
+    props.isMessageEdited = true;
+    renderer.render(
+      <ActivityItemBase {...props}>
+        {child}
+      </ActivityItemBase>
+    );
+    const component = renderer.getRenderOutput();
+
+    expect(component).toMatchSnapshot();
+  });
+
+  it('renders properly when message is not edited', () => {
+    props.isMessageEdited = false;
+    renderer.render(
+      <ActivityItemBase>
+        {child}
+      </ActivityItemBase>
+    );
+    const component = renderer.getRenderOutput();
+    
+    expect(component).toMatchSnapshot();
+  });
```

Comment:
Shouldn't we already have a test case for this? A regular test case? We shall cover this as part of that case instead.

Reply:
@mkesavan13, We do not have edit message feature supported yet in space widget. So we have added a edit message functionality when the message is edited from Web/Desktop Client. So, there is no test case related to edit.

Reply:
I mean, 'renders properly when message is not edited' is the case, that is like the default message that comes into the widget right?